server/app/services/applicant/ApplicantData.java

```diff
@@ -149,4 +149,15 @@ public ImmutableMap<Path, String> getFailedUpdates() {
   public boolean updateDidFailAt(Path path) {
     return getFailedUpdates().containsKey(path);
   }
+
+  public Optional<String> getDOB() {
```

Comment:
suggestion (blocking): follow the style guide for method naming
```suggestion
  public Optional<String> getDob() {
```